src/xdocs/config_imports.xml

```diff
@@ -603,11 +603,26 @@ import android.*;
           </tr>
           <tr>
             <td>illegalPkgs</td>
-            <td>packages to reject</td>
+            <td>Packages to reject, by default checks if import is the part of package. If <b>regexp</b> variable is set, then list of packages will be
```

Comment:
`by default` is a little misleading, as you don't specify this is the default of the `regexp` property.
Please use same wording as next sentence: `If <b>regexp</b> variable is ...`.